libs/design/src/molecules/sidebar/scss/helpers.scss

```diff
@@ -0,0 +1,6 @@
+// Stacking Context Layers
```

Comment:
I think this file can be moved in the helpers folder and renamed as `variables.scss`

Reply:
9aeac18417d8e89e68819b8465ccb04ea84cde01

Reply:
9aeac18417d8e89e68819b8465ccb04ea84cde01